src/main/webapp/default-site/scripts/rest/api/1/security/validate-session.get.groovy

```diff
@@ -16,18 +15,24 @@
  */
 
 import scripts.api.SecurityServices
+import scripts.api.UserServices
 
 def result = [:]
 
 def context = SecurityServices.createContext(applicationContext, request)
+
 try {
     def success = SecurityServices.validateSession(context, request)
     if (success) {
         result.message = "OK"
+        result.active = true
+        result.user = UserServices.getCurrentUser(context)
         response.setStatus(200)
     } else {
-        result.message = "Unauthorized"
-        response.setStatus(401)
+        result.message = null
+        result.active = false
+        result.user = null
+        response.setStatus(200)
```

Comment:
For consistency, I would say response status should be 401. Is there any reason why now we're sending 200?

Reply:
The response format is changed as a new feature is added in UI to support SAML session expired